### DISCUSSION TITLE
DoRA Support.

### DISCUSSION THREAD

**Discussion by elen07zz** [lllyasviel/stable-diffusion-webui-forge#2131] [General]
Hello, I'm curious if this fork has DoRA support.

**Comment by wktra** [comment]
It's a trivial fix with an available commit! Add it already! @lllyasviel

**Reply by Haoming02** [reply]
It already does, no?

https://github.com/lllyasviel/stable-diffusion-webui-forge/blob/main/backend/patcher/lora.py#L52
